src/main/webapp/contact.html

```diff
@@ -1,29 +1,104 @@
 <!DOCTYPE html>
 <html>
   <head>
-      <meta charset="utf-8" />
-      <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
-      <title>contact</title>
-      <!-- css -->
-      <link type="text/css" rel="stylesheet" href="./assets/css/mainStyle.css">
-      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css">
-      <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
-      <!-- js -->
-      <script src="./assets/js/mainScript.js"></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+    <title>contact</title>
+    <!-- css -->
+    <link type="text/css" rel="stylesheet" href="./assets/css/mainStyle.css">
+    <link type="text/css" rel="stylesheet" href="./assets/css/contactStyle.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <!-- js -->
+    <script src="./assets/js/mainScript.js"></script>
   </head>
   <body>
     <!-- nav bar -->
     <header>
       <div class="logo"><a href="/">noms</a></div>
       <ul class="nav">
-          <li class="navlink"><a href="/about.html">about</a></li>
-          <li class="navlink"><a href="/contact.html">contact</a></li>
-          <li class="navlink"><a href="/sign-up.html">sign up</a></li>
+        <li class="navlink"><a href="/about.html">about</a></li>
+        <li class="navlink"><a href="/contact.html">contact</a></li>
+        <li class="navlink"><a href="/sign-up.html">sign up</a></li>
       </ul>
       <div class="burger">
-          <i class="fas fa-bars"></i>
+        <i class="fas fa-bars"></i>
       </div>
     </header>
-    <h1 class="title">contact page to be added</h1>
+    <!-- main content -->
+    <div id="col-container">
+      <h1 id="title">contact</h1>
+        <div id="page-container">
+          <div id="map-container">
+            <div id="map">
+            </div>
+          </div>
+          <div id="podmates">
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/uma.jpeg" alt="Picture of Uma">
+                  <div class="col">
+                    <h4>Uma Bahl | Santa Clara University | c/o 2022</h4>
```

Comment:
Just out of curiosity, what does c/o mean in this context? I could only find https://en.wiktionary.org/wiki/c/o

Reply:
It means class of!

Reply:
Oh that makes sense! I just hadn't seen that before :)